Rakefile

```diff
@@ -0,0 +1,4 @@
+require 'bundler/gem_tasks'
```

Comment:
Because I like to `bundle exec rake build` and `bundle exec rake install`.

We can remove this if you want to.